test/config.yaml

```diff
@@ -0,0 +1,86 @@
+#### General Settings ####
+# location of system level settings
+settings:
+  title: My Very Cool Project
+  Author: Bob
+
+  # path like settings
+  data: /data/bob/original_data
+
+  # conda environment names
+  python2: py2.7
+
+  # names to access specific envs by
+  env: HOME
+
+#### Experiment Level Settings ####
+# experiment level settings, settings that apply to all samples
+exp.settings:
+
+  # Sample information relating sample specific settings to sample ids
+  sampleinfo: sample_metadata.csv
+
+  # it would be nice to be able define a setting here that applies to all
+  # samples, or define for each sample in the sampleinfo table case they are
+  # different.
+  fastq_suffix: '.fastq.gz'
+  # Need to some way to specify annotation to use, maybe here is not the best
+  # place.
+  annotation:
+    genic: /data/...
+    transcript: /data/....
+    intergenic: /data/...
+  # add modeling information here
+  models:
+    formula: ~ sex + tissue + time
+    # tell which columns in sample table should be treated like factors
+    factors:
+      - sex
+      - tissue
+      - time
```

Comment:
I think the annotations and models should go in workflow.rnaseq. Also I find myself using different models. This could be implemented by having multiple models and using jinja-templated R scripts